src/mainboard/amd/romecrb/src/msr.rs

```diff
@@ -0,0 +1,428 @@
+use core::fmt::Write;
+use x86_64::instructions::{rdmsr, wrmsr};
+/// Only read the MSR
```

Comment:
Please put a blank line before this line.

Reply:
should we follow the go style of doing this: 
/// read_msr reads an msr, compares it to a "known good" value, and prints a message, indicating
if matches or is different from that value.